src/pages/SykepengerBulk.tsx

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 import 'nav-frontend-tabell-style';
-import { FormContext, useForm } from 'react-hook-form';
+import {FormContext, useForm} from 'react-hook-form';
 import '@navikt/bedriftsmeny/lib/bedriftsmeny.css';
-import './SykepengerBulk.less';
 import Ansatte from '../components/ansatte/Ansatte';
 import InnloggetSide from './InnloggetSide';
-import { CoronaTopptekst } from '../components/CoronaTopptekst';
+import {CoronaTopptekst} from '../components/CoronaTopptekst';
```

Comment:
Her er det to autoformateringer som jobber mot hverandre, kanskje?

Reply:
Ja. Tross av at jeg har satt opp eslint i intellij så blir ikke koden riktig formattert. Har veldig lyst til å få enablet riktig formattering på tvers av utviklingsverktøy.